frb/associate/frbs.py

```diff
@@ -27,7 +27,7 @@
 #Notes:
 #"""
 updates = dict(
-    name='FRB190608',
+    name='FRB20190608',
```

Comment:
Do we want to add the end letters as well for the naming (to be completely consistent)?

Reply:
yes, I think someday.  But not this time.